api/app/GraphQL/Mutations/SubmitApplication.php

```diff
@@ -21,7 +21,7 @@ public function __invoke($_, array $args)
     {
         // grab the specific application
         // submit to validator the PoolCandidate model
-        $application = PoolCandidate::find($args['id']);
+        $application = PoolCandidate::with(['screeningQuestionResponses'])->find($args['id']);
```

Comment:
I think this is required but not 100% sure 🤔

Reply:
It works OK for me without it.  Were you getting lazy-load errors?